core/server/data/migrations/versions/4.4/01-add-oauth-user-data.js

```diff
@@ -0,0 +1,17 @@
+const {
+    addTable,
+    combineNonTransactionalMigrations
+} = require('../../utils');
+
+module.exports = combineNonTransactionalMigrations(
```

Comment:
You can just do:

```suggestion
module.exports = addTable('oauth', {
```

Reply:
yess good point 💯 , it's code from the previous implementation!